include/elm/inhstruct/DLList.h

```diff
@@ -31,31 +31,38 @@ class DLNode {
 	friend class DLList;
 	DLNode *nxt, *prv;
 public:
+	DLNode() : nxt(nullptr), prv(nullptr) {}
```

Comment:
This change probably does nothing either, compilers tend to zero-initialise this stuff (not sure if it's C++ standard actually)